PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx

```diff
@@ -63,7 +63,10 @@ function MasterControl:Init()
 			if not humanoid then return end
 			
 			if humanoid and not humanoid.PlatformStand and isJumping then
-				humanoid.Jump = isJumping
+				local state = humanoid:GetState()
+				if state ~= Enum.HumanoidStateType.Jumping and state ~= Enum.HumanoidStateType.Freefall and state ~= Enum.HumanoidStateType.Landed then
```

Comment:
Does this work for games that try to read the Jump value again while the humanoid is in the air to do a double jump? I think Super Bomb Survival does this.